code-challenges.test.js

```diff
@@ -65,3 +125,25 @@ const accountTransactions3 = []
 // Expected output: []
 
 // b) Create the function that makes the test pass.
+
+// create function accumulatingArray
+// create if statement if array is empty to return empty array using arr.length strictly equal to 0 to return [], and an else for the rest
+// create new let conditional with an array holding first value [0]
+// create for loop to iterate after the first value i = 1 
+// sum the value of newArray
+// return newArray
+// realize that I should've named accumulatingArray something shorter and easier to type 😐 (I suppose I could change it now)
```

Comment:
Lol, I am all about detailed naming conventions rather than shorter.  If your naming conventions provide detail and context, this helps future you and others that you collaborate with to know what they are looking at.